src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -79,4 +81,9 @@ export class MdTabLink {
     '[class.mat-tab-link]': 'true',
   },
 })
-export class MdTabLinkRipple extends MdRipple {}
+export class MdTabLinkRipple extends MdRipple {
+  constructor(elementRef: ElementRef, ngZone: NgZone, ruler: ViewportRuler,
+              @Optional() @Inject(MD_DISABLE_RIPPLES) forceDisableRipples: boolean) {
+    super(elementRef, ngZone, ruler, forceDisableRipples);
+  }
```

Comment:
Tried replicating the issue in our tests without any luck. Actually surprised the tests were working before since it appears they would have had issues.